apps/test/unit/applab/ImportScreensDialogTest.js

```diff
@@ -39,6 +39,25 @@ describe("AssetListItem", () => {
       'Warning: Importing this will replace your existing "bar.mp3".'
     );
   });
+
+  it("uses the current project id when no other id is specified", () => {
+    const itemWithoutProjectId = mount(
+      <AssetListItem
+        asset={{filename: 'bar.png', category: 'image', willReplace: false}}
+      />
+    );
+    expect(itemWithoutProjectId.find('img').prop('src')).to.contain("/v3/assets/undefined/bar.png");
```

Comment:
Nonblocking: Is there a way to assert what the "current project id" is in this test?

Reply:
I don't believe so. Since the clientApi.js refers to the project.js to get the project name and this AssetListItem lives independently of a project, I don't believe there is a way to associate this AssetListItem with a project.